cicecore/cicedyn/general/ice_forcing.F90

```diff
@@ -2237,30 +2238,42 @@ subroutine JRA55_files(yr)
       ! cnt represents the possible file format options and steps thru them until one is found
       exists = .false.
       cnt = 1
-      do while (.not.exists .and. cnt <= 6)
+      do while (.not.exists .and. cnt <= 8)
+
+         ! search for jra55 file with creation date (jra55date) in filename first
          if (cnt == 1) uwind_file = trim(atm_data_dir)//'/'//trim(atm_data_type_prefix)// &
+                                    '/8XDAILY/'//trim(atm_data_type_prefix)//'_'//trim(atm_data_date)//'_'// &
```

Comment:
We still have the problem of double underscore if atm_data_date is empty.  This should be

 '/8XDAILY/'//trim(atm_data_type_prefix)//trim(atm_data_date)//'_'// &

and atm_data_date should have a leading _ if that's part of the filename prefix.